packages/device_info/example/lib/main.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:device_info/device_info.dart';
+
+void main() {
+  runZoned(() {
```

Comment:
In order to capture both sync and async errors with one error handler. Also, I found that on iOS (at least on the Simulator), if the app crashes (roughly) during instantiation the stack trace is missing from STDOUT.